code/modules/clothing/clothing.dm

```diff
@@ -292,6 +297,80 @@
 			siemens_coefficient = between(0, material.conductivity / 10, 10)
 		slowdown = between(0, round(material.weight / 10, 0.1), 6)
 
+/obj/item/clothing/proc/get_accessory(var/typepath)
+	if(istype(src, typepath))
+		return src
```

Comment:
This is in case an accessory is worn on the uniform/suit slot, rather than on clothing. Ponchos & taj cloaks are quick examples of stuff that allows this. The rest of the proc is to get the specific accessory within the item, as shown with holsters.